lambda_code/scan/scan.py

```diff
@@ -26,6 +27,11 @@
 env_name = os.environ["ENVIRONMENT"]
 production_env = os.environ["PRODUCTION_ENVIRONMENT"]
 
+# Brightcove AWS accounts that are out of scope for domain-protect
+bc_acct_blacklist = [
```

Comment:
@paulschwarzenberger is it worth it to add a deny list of account ids directly into the lambda ? If so, then the user can provide a list of account ids as a parameter to ignore with custom code

Reply:
The lambdas get installed to the security account, you could simply not deploy the Domain Protect audit role to certain accounts, then Domain Protect wouldn't scan those accounts, and would include a warning message in the Lambda logs.

Reply:
I've spoken to some people that have deployed domain protect and the common method of deploying the roles across the org is by using an org stackset which is an all or nothing deployment for each account. 

I suppose those stacks could be modified to exclude some accounts but cloudformation doesn't make that very easy. 🤔